app/gui2/src/stores/project/computedValueRegistry.ts

```diff
@@ -44,8 +44,7 @@ export class ComputedValueRegistry {
   processUpdates(updates: ExpressionUpdate[]) {
     for (const update of updates) {
       const info = this.db.get(update.expressionId)
-      if (info) updateInfo(info, update)
-      else this.db.set(update.expressionId, combineInfo(undefined, update))
+      this.db.set(update.expressionId, combineInfo(info, update))
```

Comment:
We actually need to talk about this, because something fishy is happening. https://discord.com/channels/401396655599124480/1201853512409939968/1240260886472425482